Doc/library/functools.rst

```diff
@@ -76,7 +76,8 @@ The :mod:`functools` module defines the following functions:
    .. versionadded:: 3.2
 
 
-.. decorator:: lru_cache(maxsize=128, typed=False)
+.. decorator:: lru_cache(user_function)
```

Comment:
```suggestion
.. decorator:: lru_cache
```
`lru_cache(user_function)` is a decorated function. `lru_cache` itself is a decorator.